.github/workflows/push-package-solarwinds.yml

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2023 SolarWinds, LLC.
+# All rights reserved.
+
+name: Ruby Gem to Github Package
+
+on:
+  workflow_dispatch:
+    inputs:
+      rubygem-path:
+        required: true
+        description: 'The path that package you want to publish (should include /)'
+        # metrics_sdk/
+        # metrics_api/
+        # exporter/otlp/
```

Comment:
Instead of needing an input, we should be able to determine the path to the package based on the gem we want to publish, right?  Taking a look at how upstream OTel Ruby does releases, it uses a `toys` gem which may not suit our needs since it seems to only support releasing to rubygems, but it does [have a defined mapping of gems and paths](https://github.com/open-telemetry/opentelemetry-ruby/blob/main/.toys/.data/releases.yml) which IMHO is a good idea.

Reply:
@xuan-cao-swi have you tried the defined mapping approach that @cheempz suggested here?

IIUC, the `sw_build_and_push_gem.sh` script finds the correct path given `GEM_NAME` and now exits with 1 if it cannot be found. I'm wondering if a defined mapping could save on errors from typos.

Reply:
Yes, I looked into the toys library but unfortunately, if I want to use toy, then I have to make modification on upstream files, which may cause file conflicts in the future when I pull from upstream. That's why I choose to explore the shell script approach. Plus, the pre-defined mapping only works for existing `GEM_NAME`, if I want to introduce a new gem, then in that case I have to change the mapping again.